index.js

```diff
@@ -105,7 +171,17 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  let moviesGenre = [];
+  let newGenre = (genre.slice(genre.charAt(0),1)).toUpperCase() + (genre.slice(1,genre.length)).toLowerCase();
+  for (let i = 0; i < movies.length; i++) {
+    if (movies[i].genre.includes(newGenre)) {
+     moviesGenre.push(movies[i]);
+    }
+  }
+  return moviesGenre;
```

Comment:
Great call on 176.  But if we want to compare strings case insensitive - why not call `toLowerCase()` on both variables?  This way we know there are zero capitals.